sos/report/plugins/container_log.py

```diff
@@ -9,26 +9,51 @@
 # See the LICENSE file in the source distribution for further information.
 
 import os
-from sos.report.plugins import Plugin, IndependentPlugin
+from sos.report.plugins import Plugin, IndependentPlugin, PluginOpt
 
 
 class ContainerLog(Plugin, IndependentPlugin):
 
     short_desc = 'All logs under /var/log/containers'
     plugin_name = 'container_log'
     logdir = '/var/log/containers/'
+    poddir = '/var/log/pods/'
     files = (logdir, )
 
+    option_list = [
+        PluginOpt('rotated', default=False, val_type=bool,
+                  desc='get all logs from /var/log/pods regardless of size, '
+                  'which will include all rotated logs'),
+        PluginOpt('maxage', default=None, val_type=int,
+                  desc='gather only logs with `mtime` not older than this many'
+                  ' hours')
```

Comment:
I think if we do anything here, we should add a `maxage` (or `since`) -esque options to all plugins by default, like we do for `timeout` and `postproc`.

Reply:
Adding that global feature in this PR doesn't feel right, so I'll submit another PR for that one.

Reply:
Does this PR look reasonable? https://github.com/sosreport/sos/pull/3681